testsuite/thermo_integration/test_thermo_integration.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import espressopp
+import mpi4py.MPI as MPI
+
+import unittest
+
+class TestThermoIntegration(unittest.TestCase):
+    def setUp(self):
+
```

Comment:
@acfogarty Ah ok, I see it! Yes, that was also my other thoughts, maybe the empty lines before and after the class definition. Otherwise if it's the imports one could even imagine to agree on a convention to group the "system" imports (sys, time, mpi4py,unittest), then space, then import all the espressopp (user developed), as it's done sometimes in C/C++ code when someone wants to point out the system includes and the user developed includes or non-system/third parties includes, otherwise I don't see a problem with the way it is right now. But he will let you know what he meant!